test/addons/async-hello-world/binding.cc

```diff
@@ -8,13 +8,25 @@
 #include <unistd.h>
 #endif
 
+using v8::Isolate;
+using v8::Persistent;
+using v8::Function;
+using v8::HandleScope;
+using v8::Local;
+using v8::Null;
+using v8::Value;
+using v8::Integer;
+using v8::TryCatch;
+using v8::Function;
+using v8::Object;
+using v8::FunctionCallbackInfo;
 
 struct async_req {
   uv_work_t req;
   int input;
   int output;
-  v8::Isolate* isolate;
-  v8::Persistent<v8::Function> callback;
+  Isolate isolate;
```

Comment:
This should still be a pointer, right? Does this addon compile this way?

Reply:
Yes I fixed it should be a pointer reference